integration-tests/bats/schema-changes.bats

```diff
@@ -254,3 +254,452 @@ SQL
     [[ "$output" =~ '>   `v2` VARCHAR(1024)' ]] || false
     [[ "$output" =~ 'PRIMARY KEY' ]] || false
 }
+
```

Comment:
nit: great test coverage here, but there are so many new ones, you should split them out into their own file 
`primary-key-change.bats`

Reply:
I actually prefer less file clutter in our bats directories. This is technically another type of schema-change.

Reply:
Aesthetically, it's subjective, but we've started parallelizing BATS by passing each file to to its own github action. It makes CI quicker to have smaller files